code/modules/antagonists/vampire/powers/levelspells.dm

```diff
@@ -0,0 +1,163 @@
+/**
+ * Given to Vampires at the start and taken away as soon as they select a clan.
+ */
+/datum/action/vampire/clanselect
+	name = "Select Clan"
+	desc = "Take the first step as a true kindred and remember your true lineage."
+	button_icon_state = "clanselect"
+	power_explanation = "Activate to select your unique vampire clan."
+	power_flags = BP_AM_SINGLEUSE
+	check_flags = BP_CANT_USE_IN_TORPOR | BP_CANT_USE_IN_FRENZY | BP_CANT_USE_WHILE_STAKED | BP_CANT_USE_WHILE_INCAPACITATED | BP_CANT_USE_WHILE_UNCONSCIOUS
+	vitaecost = 0
+	cooldown_time = 5 SECONDS
+
+/datum/action/vampire/clanselect/activate_power()
+	. = ..()
+	vampiredatum_power.assign_clan_and_bane()
+	deactivate_power()
```

Comment:
What happens if you press this multiple times and then assign the clans, does it handle that or does it pick multiple times?